commands/web/xb-composer-reqs

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+## #ddev-generated
+## Description: Adds nice dependencies you want to have for e.g. running tests.
+## Usage: xb-composer-reqs
+## Example: ddev xb-composer-reqs.
+## ExecRaw: false
+## Flags: []
+## AutocompleteTerms: []
+
+cd "$DDEV_COMPOSER_ROOT/$DDEV_DOCROOT" || exit 1
+
```

Comment:
Now we can use

`ddev . composer -d /var/www/html/web/modules/contrib/canvas run install-dev-deps`

and will be consistent with those not using ddev. We should still provide an easier command here.